Technical-Steering-Committee/issues.md

```diff
@@ -0,0 +1,49 @@
+# Issues
+
+All the squads within the Zowe projects use GitHub to track the issues raised against the projects
+they are responsible for. The issues could be questions, bug reports, request for enhancements and others.
+Every squad responds to the new issues within the 90 days of the creation of the issue. 
```

Comment:
IMO, this period is too relaxed, especially if the issue is a critical defect or a security vulnerability. I think all squads should have a regular triaging process that occurs at least once per sprint where priority/severity of the issues are defined. Then the time frame of reaction for each priority/severity could be different and more relaxed for the low severity issues. 

I see later in the document this practice being described so I would suggest to omit the 90-days statement here to avoid wrong assumptions right in the beginning.

Reply:
This is coming from Golden CII Badge requirements and therefore this is actually auditable time frame we have to follow. As such it belongs here. We may move the statement down, but it should be here.

Reply:
I would propose it to move it after the part where want to establish as a common practice and maybe add "_latest_ within 90 days" so that we are more clear that we would want to bind us to higher standards.

Reply:
Moved below.